scripts/run

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# run the service in one of different deployment modes
+set -e
+
+usage() {
+  echo "usage: $0 [-h|--help] LOCATION"
+  echo ""
+  echo "[-h|--help]      print this help text"
+  echo ""
+  echo "LOCATION is one of:"
+  echo "- local            run as a local process"
+  echo "- docker           run in a docker container"
+}
+
+# script variables and defaults
+source $(dirname $0)/init.sh
+
+
+# process command-line options (if any)
+die() { log_error "$*" >&2; echo ""; usage; exit 2; }  # complain to STDERR and exit with error
+needs_arg() { if [ -z "$OPTARG" ]; then die "No arg for --$OPT option"; fi; }
+
+while getopts h-: OPT; do
+  # support long options: https://stackoverflow.com/a/28466267/519360
+  if [ "$OPT" = "-" ]; then   # long option: reformulate OPT and OPTARG
+    OPT="${OPTARG%%=*}"       # extract long option name
+    OPTARG="${OPTARG#$OPT}"   # extract long option argument (may be empty)
+    OPTARG="${OPTARG#=}"      # if long option argument, remove assigning `=`
+  fi
+  case "$OPT" in
+    h | help )     usage; exit 0 ;;
+    \? )           usage; exit 2 ;;  # bad short option (error reported via getopts)
+    * )            die "Illegal option --$OPT" ;;            # bad long option
+  esac
+done
+shift $((OPTIND-1)) # remove parsed options and args from $@ list
+
+
+# process positional arguments (if any)
+LOCATION=${1}
+
+run_local() {
+  APP_LOGS="${TMP_DIR}/application.log"
+  echo "Application starting. Logs will be located at: ${APP_LOGS}".
+  cd "$ROOT_DIR"
+  ./gradlew :service:bootRun | tee ${APP_LOGS}
+}
+
+run_docker() {
+  # "$ROOT_DIR/scripts/build" docker
+  # docker compose -f "$ROOT_DIR/scripts/compose.yaml" up service -d
+  echo "Running in docker is not yet implemented"
```

Comment:
Running in docker hasn't been implemented (as far as I know).

Reply:
Hmmm, I'm still having difficulty installing the build into docker. But I have minimal docker knowledge, and haven't used `docker compose`.

@sergiygetlin have you built the docker image locally (I can do this successfully) and then run it in docker? If so, what command do you use to install/start the service in docker?

Reply:
... for reference, the error I got when trying to run it with this command:
```shell
docker run --rm terra-landing-zone-service:88405f3
```

is this:
```shell
Caused by: java.lang.ClassNotFoundException: bio.terra.lz.futureservice
```

Reply:
@cahrens 
The issue with ClassNotFoundException has been addressed already in this [service/gradle/publishing.gradle](https://github.com/DataBiosphere/terra-landing-zone-service/pull/415/files#), but it is not merged yet. So steps to run service in docker:
1) please make sure that mainClass = 'bio.terra.lz.futureservice.app.LandingZoneApplication' in publishing.gradle
2) ./gradlew :service:jibDockerBuild --image=gcr.io/broad-dsp-gcr-public/terra-landing-zone-service:lz-test -Djib.console=plain
3) docker run --rm gcr.io/broad-dsp-gcr-public/terra-landing-zone-service:lz-test

Reply:
it looks like you have thumbs on the PR ... is there anything else preventing you from merging?

Reply:
@cpate4 I am waiting for you to merge your java-template-repository PR, and then I will go back to this and clean it all up so it matches that. 

(Perhaps you commented on the wrong PR?)

Reply:
@cahrens - nope 😉 . my question was directed at @sergiygetlin 's comment re: "the ClassNotFoundException has been addressed already in this ..."

Reply:
I'll merge my PR soon - I am Team Hero this week and regretting life decisions - 🤪

Reply:
I thought I responded to this ... 😢 

@cahrens - yes, the comment was intended for @sergiygetlin related to his, "... ClassNotFoundException has been addressed already in this [PR]"-comment. Just curious if there was anything preventing it from being merged. 

As for the `terra-java-project-template`, I will merge that this afternoon (trying to knock out some Team Hero stuff)

Reply:
> @cahrens - nope 😉 . my question was directed at @sergiygetlin 's comment re: "the ClassNotFoundException has been addressed already in this ..."

@cpate4 yes this fix has been merged today.